wellregistry/registry/admin/monitoring_location.py

```diff
@@ -16,199 +16,202 @@
 from .bulk_upload import BulkUploadView, BulkUploadTemplateView
 from .fetch_from_nwis import FetchFromNwisView
 from .auto_complete import SiteNoAutoCompleteView
+from django.db import connection
```

Comment:
it might be better to have this django import with the others.

Reply:
fixed